src/Symfony/Bundle/FrameworkBundle/Tests/DependencyInjection/FrameworkExtensionTest.php

```diff
@@ -566,19 +565,17 @@ public function testSerializerCacheDisabled()
 
     /**
      * @group legacy
-     * @requires function Symfony\Bridge\PhpUnit\ErrorAssert::assertDeprecationsAreTriggered
+     * @expectedDeprecation The "framework.serializer.cache" option is deprecated %s.
```

Comment:
Before, this test wouldn't have been executed, when the PhpUnitBridge was too old. Is there any risk in always running tests now (except that the expected report may be wrong)?